js/src/rive_advanced.mjs.d.ts

```diff
@@ -71,95 +147,223 @@ interface RiveOptions {
+   *
+   * Important: Make sure to delete this instance when it's no longer in use via the `delete()` method. This deletes the underlying c++ reference and
+   * frees up the backing WASM object. This can be done in cases where the user navigates away from the page with this animation, the canvas is unmounted, etc.
+   */
   export declare class Artboard {
+    /**
+     * Get the name of this Artboard instance
+     */
     get name(): string;
+    /**
+     * Get the bounds of this Artboard instance
+     */
     get bounds(): AABB;
     get frameOrigin(): boolean;
     set frameOrigin(val: boolean);
-    // Deletes the backing wasm artboard instance
+    /**
+     * Deletes the underlying instance created via the WASM. It's important to clean up this instance when no longer in use
+     */
     delete(): void;
+    /**
+     * Advances the Artboard instance by the set amount of seconds. This method updates each object in the Artboard with any changes that animations apply on properties of the objects.
+     * This should be called after calling `advance()` of a LinearAnimationInstance or StateMachineInstance
+     * @param sec - Scrub the Artboard instance by a number of seconds
+     */
     advance(sec: number): any;
+    /**
+     * Draws the artboard with a given rendering context.
+     * @param renderer - Renderer context to draw with
+     */
     draw(renderer: CanvasRenderer): void;
+    /**
+     * Creates a LinearAnimationinstance for the animation with the given name
+     * @param name - Name of the animation to create an instance for
+     * @returns A new LinearAnimationInstance object
+     */
     animationByName(name: string): LinearAnimationInstance;
+    /**
+     * Creates a LinearAnimationinstance for the animation with the given index
+     * @param index - Index of the animation to create an instance for
+     * @returns A new LinearAnimationInstance object
+     */
     animationByIndex(index: number): LinearAnimationInstance;
+    /**
+     * Returns the number of animations in the artboard
+     * @returns Number of animations on the Artboard
+     */
     animationCount(): number;
+    /**
+     * Creates a StateMachineInstance for the state machine with the given name
+     * @param name - Name of the state machine to create an instance for
+     * @returns A new StateMachineInstance object
+     */
     stateMachineByName(name: string): StateMachineInstance;
+    /**
+     * Creates a StateMachineInstance for the state machine with the given index
+     * @param index - Index of the state machine to create an instance for
+     * @returns A new StateMachineInstance object
+     */
     stateMachineByIndex(index: number): StateMachineInstance;
+    /**
+     * Returns the number of state machines in the artboard
+     * @returns Number of state machines on the Artboard
+     */
     stateMachineCount(): number;
+    /**
+     * Returns a reference for a Bone object of a given name
```

Comment:
If I don't know how a bone works, is there somewhere I can jump to read up on it?

Reply:
Honestly not much functionality here that we expose, beyond coordinates and a length. But we can maybe link to the Bone feature in the editor (our help docs center)